### PR TITLE
add earthquake scenario

### PR DESCRIPTION
**Pull request by Gustry**
### What does it fix ?
* Tickets: #3666 #3719 #3803 #3732
* Funded by : DFAT
* Description : Fix EQ IF

Known issue : breakdown exposed, not exposed, affected in the report

### CODE REVIEW

File: safe/impact_function/test/test_impact_function.py [hunk 1]
```diff
@@ -374,6 +374,7 @@ def test_scenarii(self):
         Let's keep booleans to False by default.
         """
         scenarii = {
```
Comment (timlinux) [inline safe/impact_function/test/test_impact_function.py]: Would still like to see this called 'scenarios' rather than 'scenarii'.

**Comment by timlinux** [comment]
LGTM when tests pass....

**Comment by Gustry** [comment]
@ivanbusthomi Can you try this PR about EQ raster on population raster ? 
The PR is not finished, but I will really appreciate if I can have some feedbacks already. Thanks a lot.

File: safe/definitions/hazard_classifications.py [hunk 2]
```diff
@@ -183,11 +182,10 @@
             'affected': True,
             'description': tr('The medium hazard class.'),
             'string_defaults': ['medium'],
-            # Not used because EQ algs take care of this
-            # 'displacement_rate': 0.0,
+            'displacement_rate': 1.0,
             # Not used because EQ algs take care of this
             # 'fatality_rate': 0.0,
-            'numeric_default_min': 7,
+            'numeric_default_min': 6,
```
Comment (Gustry) [inline safe/definitions/hazard_classifications.py]: @Charlotte-Morgan You changed these values yesterday. But I need to rollback, the minimum is not included. Please confirm this is still correct.

**Pull request merged**